ansible/spatial-hub.yml

```diff
@@ -0,0 +1,12 @@
+#vagrant ssh-config | grep IdentityFile
```

Comment:
Not clear what this comment is meant to be used for. The private key used to connect is irrelevant to this playbook that can be reused for non-vagrant scenarios